cmd/ipfs/main.go

```diff
@@ -106,6 +107,7 @@ func mainRet() (exitCode int) {
 		}
 	}()
 	otel.SetTracerProvider(tp)
+	otel.SetTextMapPropagator(autoprop.NewTextMapPropagator())
```

Comment:
I would like to understand what this does, because this isn't setuped yet, however http propagation already works for cmds.

Reply:
@Jorropo https://pkg.go.dev/go.opentelemetry.io/contrib/propagators/autoprop@v0.40.0#pkg-overview, for more info ask @guseggert since he added this.

Reply:
AFAIK, the main goal is that it parses the ENV variable and has sensible defaults.

Reply:
However, it might not make sense since we always use `tracing.Propagators()` explciitly, so we're never using the default.

Reply:
If we don't use it, I would say let's not do this. Deadcode is bad.

Reply:
I'm a little hazy here because this was so long ago, but presumably I was trying to make the propagation user-configurable with OTel standard environment variables. I wouldn't say this is dead code as the intention is to allow users to wire up OTel in their applications, although this could use some tests.